api/mod/src/models/annotation.py

```diff
@@ -91,6 +91,14 @@ def save(self) -> None:
         if not self.hash_is_valid():
             raise HashMismatch()
         annotation_file = SETTINGS.data_folder.joinpath(f"{self.src}.annotation.json")
+
+        trial = 2
+        while annotation_file.exists():
```

Comment:
Just one short question: Could we also try to "read" the current number from the filename (if it exists) and then instantly set the correct one instead of the while loop?

Reply:
I don't understand what you mean tbh...